### PR TITLE
Navio Release 20200922

### PR DESCRIPTION
**Pull request by corteccia**
- Update link
- Update ardupilot's versions
- Update changelog

It is also necessary to merge #342

### CODE REVIEW

File: docs/autopilots/common/dev/changelogs.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+- ArduPilot updated:
+    - ArduCopter: 4.0.3
+    - ArduPlane: 4.0.5
+    - ArduSub: 4.0.1
+
+  - Kernel updated to v4.19.27
+  - Preinstalled ROS Noetic (Buster only)
+  - Default board is Navio2
+  - Raspberry Pi 4 8GB support
+  - Fixed Dronekit connection issues
```
Comment (AlexeyBulatov) [inline docs/autopilots/common/dev/changelogs.md]: Shift it to left

File: docs/autopilots/common/dev/changelogs.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
+*2020-09-22*
+
+- ArduPilot updated:
+    - ArduCopter: 4.0.3
+    - ArduPlane: 4.0.5
+    - ArduSub: 4.0.1
+
+  - Kernel updated to v4.19.27
+  - Preinstalled ROS Noetic (Buster only)
+  - Default board is Navio2
```
Comment (AlexeyBulatov) [inline docs/autopilots/common/dev/changelogs.md]: Maybe we can rewrite this change?

File: docs/autopilots/common/dev/changelogs.md [hunk 3]
```diff
@@ -1,5 +1,18 @@
 ## Navio Raspbian changelog
 
+*2020-09-22*
+
+- ArduPilot updated:
+    - ArduCopter: 4.0.3
+    - ArduPlane: 4.0.5
+    - ArduSub: 4.0.1
+
+  - Kernel updated to v4.19.27
```
Comment (AlexeyBulatov) [inline docs/autopilots/common/dev/changelogs.md]:
```suggestion
  - Kernel updated to v4.19.127
```

File: docs/autopilots/common/dev/changelogs.md (hunk 2 above)
Comment (corteccia) [reply docs/autopilots/common/dev/changelogs.md]: @AlexeyBulatov Does it better now? I don't really know how to say it more shortly and clearly

**Review by TatianaAndreeva — APPROVED** [review]

**Review by AlexeyBulatov — APPROVED** [review]

**Pull request merged**